static/app/views/projectInstall/createProject.tsx

```diff
@@ -373,11 +329,20 @@ export function CreateProject() {
         }
       }
     },
-    [createRules, organization, createProject, setCreatedProject, navigate, api]
+    [
+      organization,
+      setCreatedProject,
+      navigate,
+      api,
+      createProjectAndRules,
+      createNotificationAction,
+    ]
   );
 
   const handleProjectCreation = useCallback(
     async (data: FormData) => {
+      setErrors(undefined);
```

Comment:
This is the only part slightly outside the scope of this PR, but I think it's reasonable to include it. We should clear the errors when the submit button is clicked again.

Reply:
I’m wondering why we keep track of the error state manually? `useMutation` handles errors and returns them as part of the return value:

```
const { error } = useMutation(...)
```

and it will get re-set automatically if you fire the mutation again. So I think we should be able to get rid of that `useState` completely.

Reply:
@TkDodo yes! great idea. now that we have all mutations in `useCreateProjectAndRules` we can get rid of `setErrors`. will update the code